apps/web/src/components/(dashboard)/common/command-menu.tsx

```diff
@@ -53,6 +62,32 @@ export function CommandMenu({ open, onOpenChange }: CommandMenuProps) {
   const [search, setSearch] = useState('');
   const [pages, setPages] = useState<string[]>([]);
 
+  // const [searchResults, setSearchResults] = useState<
+  //   { label: string; path: string; value: string }[]
+  // >([]);
+
+  const { data: searchDocuments, isLoading: isSearchingDocuments } =
+    trpcReact.document.searchDocuments.useQuery(
+      {
+        query: search,
+      },
+      // {
```

Comment:
We should add `keepPreviousData` so we can hold onto the old results while we're fetching 😄